.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,8 @@
+blank_issues_enabled: false
+contact_links:
+  - name: Supports and troubleshooting
+    url: https://github.com/sequelize/sequelize/discussions
+    about: This issue tracker is not for bug reports which requires a SSCCE. Please refer to the bug report issue template.
```

Comment:
The issue tracker is for all bug reports and feature requests. If a bug report can't be replicated in a SSCCE, it should still go in the tracker

I propose to rename the link "Help & Questions" and to change its description to "The issue tracker is for bug reports and feature requests. Use GitHub Discussions for question and help requests"?

Reply:
That's better